packages/flutter/test/services/text_input_test.dart

```diff
@@ -375,7 +376,9 @@ void main() {
       expect(TextInputType.visiblePassword.index, 7);
       expect(TextInputType.name.index, 8);
       expect(TextInputType.streetAddress.index, 9);
+      expect(TextInputType.streetAddress.index, 9);
```

Comment:
nit: I don't think we need another check for street address after the first one.